modules/shared/src/main/scala/org/tessellation/implicits/trust.scala

```diff
@@ -0,0 +1,17 @@
+package org.tessellation.implicits
```

Comment:
you should not mix test with other code. test-specific code should be in test-related modules (see where other generators are)

Reply:
Moved the implicits into the same object as the other generators.